setup.py

```diff
@@ -4,7 +4,7 @@
     long_description = fh.read()
 
 setup(
-    name="nso_ds_classes",  # Replace with your own username
+    name="satellite_images_nso_datascience",  # Replace with your own username
     version="1.0.0",
```

Comment:
Should we not update this version? Also doesn't this name change break the import statements in all the other repo's that rely on this package?

Reply:
only this repo uses these packages since the export of this repo is a .sav file.

tests proves it works as well

Reply:
All right, you're correct on the repo usage in other repo's. We should still change the version in my opinion, because we have made changes to the package. (1.0.1 or something is fine).

Reply:
should be fixed